src/tests/test_plot.py

```diff
@@ -113,18 +113,19 @@ def test_summary_stats_missing_file(chinook_db, ip_empty):
 
 
 @pytest.mark.parametrize(
-    "cell, error_type, error_message",
+    "cell, error_",
```

Comment:
why is this still named error_ ?
You need to assert that the output is a matplotlib figure. There are some tests in `test_magic_plot.py`

Reply:
made the change so that we assert that the output is a matplotlib fig and there is no error.